meta-balena-common/recipes-core/initrdscripts/files/migrate

```diff
@@ -96,7 +101,9 @@ migrate_enabled() {
 migrate_run() {
     # Find the raw image in the rootfs partition
     image=$(find "${ROOTFS_DIR}" -xdev -type f -name "${BALENA_IMAGE}")
-    kernel_images=$(find "${ROOTFS_DIR}" -xdev -type f -name "@@KERNEL_IMAGETYPE@@*")
+    kernel_images=$(find "${ROOTFS_DIR}" -xdev -type f -name "Image*")
```

Comment:
@@KERNEL_IMAGETYPE@@* is replaced at build time by a Yocto variable - `Image*` does not cover all possible images like bzImage and others.

Why is replacing KERNEL_IMAGETYPE not working?

Reply:
thanks @alexgg , this change escaped me and isn't actually necessary.